drivers/wifi/nrf_wifi/Kconfig.nrfwifi

```
@@ -99,6 +99,15 @@ config NRF70_AP_MODE
 	depends on WIFI_NM_WPA_SUPPLICANT_AP
 	default y if WIFI_NM_WPA_SUPPLICANT_AP
 
+config NRF70_ENABLE_DUAL_VIF
+	bool "Dual virtual Wi-Fi interfaces"
+	default y if WIFI_NM_MAX_MANAGED_INTERFACES = 2
+	depends on WIFI_NRF7002 && NET_L2_ETHERNET
```

```suggestion
	depends on NET_L2_ETHERNET
```
SYSTEM_MODE implies nRF7002 and nRF7001, and this feature can work with both. We need to fix other modes to remove this dependency and move to SYSTEM_MODE definition.

---

Are you sure that nrf7001 supports dual vif?
I have tested with just nrf7002